e2e-test/src/util.js

```diff
@@ -1,2 +1,18 @@
-/* eslint-disable import/prefer-default-export */
+import { vBtn, vList } from 'jest-puppeteer-vuetify';
+
 export const { CLIENT_URL } = process.env;
+
+export async function deleteAllDatasets() {
+  // firstUploadedItem returns a uploaded file node list
+  const firstUploadedItem = await page.$x(vList({ content: 'upload-list theme--light' }));
+  if (firstUploadedItem.length !== 0) {
+    await expect(page).toClickXPath(vBtn('clear all'));
+    await page.waitForXPath(vBtn('Delete'));
+    await expect(page).toClickXPath(vBtn('Delete'));
+  }
+}
+
+export async function allCleared() {
+  const firstUploadedItem = await page.$x(vList({ content: 'upload-list theme--light' }));
```

Comment:
This wasn't working. Almost though:
```suggestion
  const firstUploadedItem = await page.$x(vList({ cssClass: 'upload-list theme--light' }));
```
Also you may need to add a second wait to ensure that everything has time to be removed.